docs/pages/access-controls/access-request-plugins/access-monitoring-rules.mdx

```diff
@@ -0,0 +1,114 @@
+  instructions](../../installation.mdx#linux) for your environment and Teleport edition.
+
+- At least one of the Teleport Access Plugins that support using being set up.
+
+- (!docs/pages/includes/tctl.mdx!)
+
+## Step 1/2. Create an Access Monitoring Rule
+
+You can define an Access Request notification rule in two ways:
+- Using the web UI dialog.
+- Creating a YAML resource file containing the definition of a rule.
+
+### Create an Access Monioting Rule via the web UI
+
+To create an Access Monitoring Rule via the web UI, first navigate to
+the Access Request page and click on `View Notification Routing Rules`.
+
+![View Notification Routing Rules](../../../img/enterprise/plugins/access_monitoring_rules/view_routing_rules.png)
+
+Then click `Create Notification Rule`.
+
+![Create AMR](../../../img/enterprise/plugins/access_monitoring_rules/access_monitoring_rule_ui.png)
+
+If a plugin that supports Access Monitoring Rule based routing is not enrolled
+the UI will prompt you to enroll one.
+
+### Create an Access Monitoring Rule using YAML resource
+
+Here is an example Access Monitoring Rule. This rule will notify `your_slack_channel`
+via the Slack plugin if an Access Request containing the role `your_role_name` is
+made.
+
+```yaml
+kind: access_monitoring_rule
+version: v1
+metadata:
+  name: your-plugin-name
+spec:
+  subjects: ['access_request']
+  condition: 'access_request.spec.roles.contains("your_role_name")'
+  notification:
+    name: 'slack'
+    recipients: ['your_slack_channel']
+```
+
+Above is an example of an Access Monitoring Rule that routes incoming Access Requests
+that are requesting the role `your_role_name` to the slack channel `your_slack_channel`.
+
+The above routing rule can be created using `tctl create your-file-name.yaml`
+from the command line.
+
+Multiple recipients can be specified in an Access Monitoring Rule.
+The condition field of the rule is set to a predicate expression defining the conditions
+under which you wish the rule to apply.
+
+Fields of the Access Request that are currently supported are
+| Field | Description |
+| --- | --- |
+| access_request.spec.roles | The set of roles requested. |
+| access_request.spec.suggested_reviewers | The set of reviewers specified in the request. |
+| access_request.spec.system_annotations | A map of system annotations on the request. |
+| access_request.spec.user | The requesting user. |
+| access_request.spec.request_reason | The request reason. |
+| access_request.spec.creation_time | The creation time of the request. |
+| access_request.spec.expiry | The expiry time of the request. |
+
+Predicate expressions used in the condition of Access Monitoring Rules must evaluate to
+either true or false.
+
+Some example values for the condition field and their possible uses.
+- `access_request.spec.user == "some_user"` - Routing based on requesting user.
+- `access_request.spec.roles.contains("some_role")` - Routing based on the requested roles.
```

Comment:
Is this correct?

I would expect it to be `contains(access_request.spec.roles, "some_role")`.